Pbi 05 assessee signup

aryafchandra opened the pull request.

aryafchandra committed:
[RED] create test for assessee dashboard

aryafchandra committed:
Delete .env

aryafchandra committed:
[RED] create test for assessee dashboard

aryafchandra committed:
Merge branch 'PBI-18-assessee_dashboard' of https://github.com/one-day-intern/one-day-intern-backend into PBI-18-assessee_dashboard

aryafchandra committed:
Merge branch 'staging' of https://github.com/one-day-intern/one-day-intern-backend into PBI-05-assessee_signup

aryafchandra committed:
[RED] create test for assessee registration

aryafchandra committed:
[GREEN] implement assessee registration

aryafchandra committed:
[CHORES] insert phonenumbers dependency to requirements.txt

johaneschristian commented:
Hello, Arya. I believe it is necessary to first resolve the branch conflicts locally. You may do this by pulling from the staging branch and perform a cherry-pick through your IDE. I believe that the Codacy issues will also be resolved after pulling, as a fix to those files are already in place in staging.
Qapla!

aryafchandra committed:
Resolve local conflict

aryafchandra committed:
[REFACTOR] move exceptions to one app

johaneschristian commented on one_day_intern/exceptions.py:
```diff
@@ -8,3 +8,22 @@ class InvalidAssignmentRegistration(Exception):
 
 class InvalidRequestException(Exception):
     pass
+
```
Thanks for grouping all exceptions under 1 file!

johaneschristian commented on users/tests.py:
```diff
@@ -819,6 +935,106 @@ def test_register_assessor_with_invalid_one_time_code(self):
+        def test_register_assessee_when_complete(self):
+            registration_data = self.registration_base_data.copy()
+            response = self.fetch_with_data(registration_data, REGISTER_ASSESSEE_URL)
+            self.assertEqual(response.status_code, OK_REQUEST_STATUS_CODE)
```
Hello, Arya! After the last commit, we are moving to use HTTPStatus instead of our own defined constant. Can you please change it in your following commits? Thanks!

johaneschristian reviewed (commented):
Hello, Arya! Thanks for implementing our register assessee endpoints. There is just a minor change that you need to do to comply with our new standards.
Regards,
Johanes Christian.

aryafchandra committed:
[REFACTOR] change from constant variable to HTTPStatus class attribute

change constant variable to HTTPStatus class attribute in users test

aryafchandra committed:
[REFACTOR] remove unused variable and whitespace

remove unused import in assessee/admin, assessee/models
remove duplicate function in users/services/utils
remove whitespace in users/test, users/views, users/services/utils

aryafchandra committed:
[REFACTOR] remove whitespace

remove whitespace in users/test

aryafchandra committed:
[REFACTOR] remove whitespace in users/test

remove unnecessary whitespace in users/test
